include/geode/mesh/builder/edged_curve_builder.h

```diff
@@ -62,13 +66,17 @@ namespace geode
          */
         index_t create_point( const Point< dimension >& point );
 
+        void copy( const EdgedCurve< dimension >& edged_curve, BuilderKey )
+        {
+            copy( edged_curve );

```

Comment:
why moving implementation in header file? To avoid BuilderKey naming?

Reply:
no reason in particular. It is so simple I added it in th header by simplicity. I can move the code If you want.

Reply:
No it's ok